batch_job/src/amazon_ec2.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/sh
```

Comment:
Ok, that makes sense. I will defer to @dthain and @btovar on whether this file and convert.pl are included in the final pull request, but I can see why its included now. Thanks for the clarification.